ee/clickhouse/queries/paths/path_event_query.py

```diff
@@ -28,10 +35,18 @@ def get_query(self) -> Tuple[str, Dict[str, Any]]:
         funnel_paths_join = ""
         funnel_paths_filter = ""
 
-        if self._filter.funnel_paths:
-            funnel_paths_timestamp = f"{self.FUNNEL_PERSONS_ALIAS}.timestamp as min_timestamp"
+        if self._filter.funnel_paths == FUNNEL_PATH_AFTER_STEP or self._filter.funnel_paths == FUNNEL_PATH_BEFORE_STEP:
+            # used when looking for paths up to a dropoff point to account for events happening between the latest even and when the person is deemed dropped off
+            funnel_window = f"+ INTERVAL {self._filter.funnel_window_interval or self._filter.funnel_window_days} {self._filter.funnel_window_interval_unit_ch()}"
```

Comment:
Since completely new feature, don't think we need to consider old funnels params?

```suggestion
            funnel_window = f"+ INTERVAL {self._filter.funnel_window_interval} {self._filter.funnel_window_interval_unit_ch()}"
```